packaging/rpm/microshift.spec

```diff
@@ -221,6 +221,18 @@ The microshift-gateway-api-release-info package provides release information fil
 release. These files contain the list of container image references used by Gateway API
 and can be used to embed those images into osbuilder blueprints.
 
+%package observability
+Summary: OpenTelemetry-Collector configured for MicroShift
+BuildArch: noarch
+Requires: microshift = %{version}
+Requires: opentelemetry-collector
+
+%description observability
+Deploys the Red Hat build of Opentelemetry-collector as a systemd service on host. MicroShift provides client
+certificates to permit access to the kube-apiserver metrics endpoints. If a user defined opentelemetry-collector exists
+at /etc/microshift/opentelemetry-collector.yaml, this config is used. Otherwise, a default config is provided. Note that
+the default configuration requires the backend endpoint be set by the user.
```

Comment:
```suggestion
certificates to permit access to the kube-apiserver metrics endpoints. If a user-defined configuration file exists
at /etc/microshift/opentelemetry-collector.yaml, this configuration is used. Otherwise, a default configuration is provided. 
Note that the default configuration requires the backend endpoint be set by the user.
```

Reply:
For the backend endpoint, should we be specific on what we expect users to set?
I mean, should we say `exporters.otlp` section must be edited by users?

Reply:
Added more specific instructions